core/src/main/java/com/ultramixer/jarbundler/AppBundleProperties.java

```diff
@@ -443,13 +446,21 @@ public String getMainClass() {
     }
 
     public void setJVMVersion(String s) {
+	    s = s.trim();
         mJVMVersion = s;
+		String foodle = s.endsWith("+") ? s.substring(0, s.length()-1)
```

Comment:
there are more cases which need to be considered here: `9`, `9-ea`, `9+`, `9.1`, `10`, `10.0`, `11`, `11*`, `11.13`, etc.

I implemented a solution for this years ago on another branch which hasn't been merged yet: https://github.com/UltraMixer/JarBundler/commit/282fab8540f6467d509b99e0fdad93c140a1f2e1

Should definately do that now...